dev/tasks/tasks.yml

```diff
@@ -1789,6 +1789,18 @@ tasks:
       r_image: r-base
       r_tag: 3.6-opensuse42
       not_cran: "TRUE"
+  
+  test-r-minimal-build:
```

Comment:
TODO: Use a different word because "minimal" already has a specific meaning (`LIBARROW_MINIMAL`) that's not this

Reply:
But it is minimal in the sense that the cpp minimal build in here is. I think it's fine.